scripts/presubmit.sh

```diff
@@ -100,20 +105,23 @@ main() {
     mkdir -p /tmp/trillian_profile
     rm -f /tmp/trillian_profile/*
 
-    i=0
+    local i=0
     for d in ${go_dirs}; do
+      # Create a different -coverprofile for each test (if enabled)
+      local coverflags=
+      if [[ ${coverage} -eq 1 ]]; then
+        coverflags="-covermode=atomic -coverprofile='/tmp/trillian_profile/profile_${i}.out'"
+      fi
+
       # Do not run go test in the loop, instead echo it so we can use xargs to
       # add some parallelism.
-      # Creating different -coverprofile files in a single xargs invocation is
-      # tricky, hence the loop is kept.
-      echo go test \
-        -covermode=atomic \
-        -coverprofile="/tmp/trillian_profile/profile_$((i++)).out" \
-        -short \
-        -timeout=5m \
-        ${goflags} "$d"
-    done | xargs -I '@' -P 10 bash -c '@'
-    cat /tmp/trillian_profile/profile_*.out > /tmp/coverage.txt
+      echo go test -short -timeout=5m ${coverflags} ${goflags} "$d"
+
+      i=$((i+1))
+    done | xargs -I '{}' -P 10 bash -c '{}'
+
+    [[ ${coverage} -eq 1 ]] && \
+      cat /tmp/trillian_profile/profile_*.out > /tmp/coverage.txt
```

Comment:
I'd prefer an explicit `if  ;then` like everywhere else.

Reply:
I tend to favor this style for single-action ifs (that's the only reason why I didn't convert the others). Since there's no hard-and-fast rule in either direction I'm keeping it.